core/modules/filter/js/filter.js

```diff
@@ -104,6 +104,39 @@ Backdrop.filterEditorDetach = function(field, format, trigger) {
   }
 };
 
+/**
+ * Provides summary text for the "Editor details" fieldset, under each CKEditor
+ * instance.
```

Comment:
I don't think mentioning CKEditor here is necessary or correct...

```suggestion
 * Provides summary text for the "Editor details" fieldset, under each textarea
 * field with a text format.
```

Reply:
Right 👍 ...but it should be "with a text editor" instead. Also, the name of the fieldset is "Formatting options". Lets fix all that.